dashboard/src/components/ChartView/LinkRenderer.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from "react";
+import { HashLink as Link } from "react-router-hash-link";
+
+const LinkRenderer: React.SFC<{}> = (props: any) => {
+  if (props.href.startsWith("#")) {
+    return <Link to={props.href}>{props.children}</Link>;
+  }
+  return <a href={props.href}>{props.children}</a>;
```

Comment:
Why don't we use the normal `Link` here from `react-router-dom` that we use elsewhere? Ah, because this `LinkRenderer` is only being used for rendering the README, so it will never contain internal application links - got it. Not sure if it's worth a comment, up to you.